README.md

```diff
@@ -1,2 +1,77 @@
 # Microverse-Capstone-Project
 The first module capstone project.
+
+![](img/demo.png)
+
+> A webpage for online bootcamp with all the contents and instructors.
+
+## Built With
+
+- HTML, CSS & JS
+- Hint, Stylelint, Eslint
+
+## Live Demo (if available)
+
+[Web Development Course](https://github.com/EzekielUtshudi/Microverse-Capstone-Project)
+
```

Comment:
- [ ] Hi, your project looks very good. I think here you put the wrong link to your deployed project. If you haven't deploy please make sure to do so and provide the link here in your readme. I've attached an small video on how to find the setting to deploy to github pages.

![pages](https://user-images.githubusercontent.com/18730106/157543073-fa57c182-9246-470a-8265-bc3a9decdd44.gif)